cypress/integration/clue/branch/student_tests/graph_tool_spec.js

```diff
@@ -1,10 +1,12 @@
 import Canvas from '../../../../support/elements/common/Canvas';
 import ClueCanvas from '../../../../support/elements/clue/cCanvas';
 import GraphToolTile from '../../../../support/elements/clue/GraphToolTile';
+import RightNav from '../../../../support/elements/common/RightNav';
```

Comment:
At this point is `RightNav` the right term here?  We no longer have any right navigation in the app.  I'm assuming this is an existing test piece that is hard to strip out but needs to be used?

Reply:
Refactoring all the RightNav elements would be fairly big so I was holding off on doing that for this PR

Reply:
That was my fear - that it's probably an entire story just to rip out the `RightNav` refererence.